src/gui/accountmanager.cpp

```diff
@@ -69,6 +75,9 @@ bool AccountManager::restore()
 
 bool AccountManager::restoreFromLegacySettings()
 {
+    qDebug() << "Migrate: restoreFromLegacySettings, checking settings group"
+             << Theme::instance()->appName();
+
     // try to open the correctly themed settings
     auto settings = Utility::settingsWithGroup(Theme::instance()->appName());
```

Comment:
Could it be worth to also check this one? Maybe also in `saveAccount*` methods, which could mean that other accounts could be lost?